src/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs

```diff
@@ -0,0 +1,172 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information. 
+// Copyright (c) .NET Foundation and Contributors
+//
+// System.ComponentModel.CultureInfoConverter test cases
+//
+// Authors:
+// 	Gert Driesen (drieseng@users.sourceforge.net)
+//
+// (c) 2008 Gert Driesen
+//
+
+using System.ComponentModel.Design.Serialization;
+using System.Globalization;
+using Xunit;
+
+namespace System.ComponentModel.Tests
+{
+    public class CultureInfoConverterTest
+    {
+        private CultureInfoConverter converter;
+
+        public CultureInfoConverterTest()
+        {
+            converter = new CultureInfoConverter();
+        }
+
+        [Fact]
+        public void CanConvertFrom()
+        {
+            Assert.True(converter.CanConvertFrom(typeof(string)));
+            Assert.False(converter.CanConvertFrom(typeof(CultureInfo)));
+            Assert.False(converter.CanConvertFrom(typeof(object)));
+            Assert.False(converter.CanConvertFrom(typeof(int)));
+        }
+
+        [Fact]
+        public void CanConvertTo()
+        {
+            Assert.True(converter.CanConvertTo(typeof(string)));
+            Assert.False(converter.CanConvertTo(typeof(object)));
+            Assert.False(converter.CanConvertTo(typeof(CultureInfo)));
+            Assert.False(converter.CanConvertTo(typeof(int)));
+            Assert.True(converter.CanConvertTo(typeof(InstanceDescriptor)));
+        }
+
+        [ActiveIssue(11611, Xunit.PlatformID.AnyUnix)]
```

Comment:
`PlatformID` has been renamed `TestPlatforms`. It would be good if it was renamed here before merging. See #12284.

e.g. `[ActiveIssue(11611, TestPlatforms.AnyUnix)]`